src/backlight/metrics/evaluation_metrics.py

```diff
@@ -43,7 +43,9 @@ def calculate_sharpe(
     """
     value = positions_or_portfolio.value.resample(freq).first().dropna()
     previous_value = value.shift(periods=1)
-    log_return = np.log((value.values / previous_value.values)[1:])
+    pl = (value.values / previous_value.values)[1:]
+    pl = pl[pl > 0]
```

Comment:
We should not filter return here.
If the value become negative, we can not calculate return.
(`v_i` in  https://en.wikipedia.org/wiki/Rate_of_return)

Reply:
When we have nagative values here, the loss exceeds its principal.

Reply:
Ok. I'll cancel modifications. To get Sharpe values, I will need to adapt parameters of principal and lots sizes so lots size are not too big compared to principal to limit moments where loss exceeds principal. Thanks.